enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -62,13 +59,13 @@ is destroyed by the container whenever the container feels it is
 necessary to do so, such as when the container is shutting down.
 
 The implementation of a Port and the container it runs in are tied. A
-JAX-RPC Service Implementation Bean always runs in a web container. A
-JAX-WS Service Implementation Bean may also run in a web container. An
-EJB Service Implementation Bean always runs in an EJB container.
+A Jakarta XML Web Services Service Implementation Bean runs in a web 
```

Comment:
this leads to "A A Jakarta ..."

Reply:
also the word "may" should be kept, so sth like: "A Jakarta XML Web Services Service Implementation Bean may run in a web container..." would be better

Reply:
yes, thanks, will make these two changes.